### PR TITLE
Landing page updates

### PR DESCRIPTION
**Pull request by joshcosta**
- Updates copy to be more concise
- Adds an actual GPA insights image
- Removes "new" pill

meta
1 reviewer
rebase

### CODE REVIEW

**Review by vikaspotluri123 — APPROVED** [review]
Can you remove the old image if it's not used?

**Comment by joshcosta** [comment]
Yes, thanks

**Comment by vikaspotluri123** [comment]
`experiment.jpg`?

File: src/index.hbs [hunk 1]
```diff
@@ -100,21 +100,21 @@ description: Better grades, together. Gradebook is the only student-built, stude
...
-						See how earning different grades across assignments will affect your average
+						See how different grades will affect your average
 					</p>
 				</div>
 			</div>
 			<div class="feature-extended">
 				<div class="feature-extended-image is-revealing">
 					<img
-						src="/static/images/index/experiment.jpg"
+						src="/static/images/index/insights.jpg"
```
Comment (ramsays) [inline src/index.hbs:110]: Remove the old `experiment.jpg` image?

File: src/index.hbs [hunk 2]
```diff
@@ -31,13 +31,13 @@ description: Better grades, together. Gradebook is the only student-built, stude
...
+			No more spreadsheets.
 		</p>
 		<p class="section-paragraph mb-0 text-center">
-			Add in the unpredictability of your courses, and you've got a recipe for chaos
+			No more confusing websites.
 		</p>
-		<p class="section-paragraph mt-0 text-center">
-			<strong>Gradebook helps bring order to your chaotic life</strong>
+		<p class="section-paragraph mb-0 text-center">
+			<strong>We'll handle the math.</strong>
```
Comment (ramsays) [inline src/index.hbs:40]: Very slogany, I like it! 👍

File: src/index.hbs (hunk 1 above)
Comment (ramsays) [reply src/index.hbs:110]: Lol didn't see @vikaspotluri123 's comment 😄

Comment (joshcosta) [reply src/index.hbs:110]: Nope we still need it

**Pull request merged**